### PR TITLE
Update chart.js to better support Tooltip Position

### PR DESCRIPTION
**Pull request by Hu-Eb**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [X] Provide a URL to documentation or source code which provides context for the suggested changes: <<https://www.chartjs.org/docs/latest/configuration/tooltip.html#position-modes>>
- [X] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Review by adripanico — APPROVED** [review]
Please use a meaningful commit message next time.

**Comment by lekoaf** [comment]
> Provide a URL to documentation or source code which provides context for the suggested changes
I'm missing the above in the PR.

**Review by lekoaf — CHANGES_REQUESTED** [review]
> Provide a URL to documentation or source code which provides context for the suggested changes

I'm missing the above in the PR.

I'm guessing it's [this](https://www.chartjs.org/docs/latest/configuration/tooltip.html#position-modes) but I'd like to make sure.

**Comment by Hu-Eb** [comment]
I changed the URL to correct one

**Review by lekoaf — APPROVED** [review]

File: types/chart.js/index.d.ts [hunk 1]
```diff
@@ -313,7 +315,7 @@ declare namespace Chart {
         callbacks?: ChartTooltipCallback;
         filter?(item: ChartTooltipItem, data: ChartData): boolean;
         itemSort?(itemA: ChartTooltipItem, itemB: ChartTooltipItem): number;
-        position?: string;
+        position?: TooltipPositionType;
```
Comment (uniqueiniquity) [inline types/chart.js/index.d.ts]: This doesn't seem to be true, given the [documentation](https://www.chartjs.org/docs/latest/configuration/tooltip.html#position-modes).
This change will cause errors for any custom defined modes from `positioners`.
@lekoaf, did I misunderstand something?

**Review by uniqueiniquity — CHANGES_REQUESTED** [review]

File: types/chart.js/index.d.ts (hunk 1 above)
Comment (Hu-Eb) [reply types/chart.js/index.d.ts]: @uniqueiniquity I changed the type to support custom positioners. But I am not sure the PR still relevant.

Comment (uniqueiniquity) [reply types/chart.js/index.d.ts]: I agree; this will just be `string` in the end. I'd suggest closing the PR.

**Pull request closed without merging**